tests/storage/test_room_search.py

```diff
@@ -269,6 +268,14 @@ def prepare(
         response = self.helper.send(self.room_id, self.PHRASE, tok=self.access_token)
         self.assertIn("event_id", response)
 
+        # The behaviour of a missing trailing double quote changed in PostgreSQL 14
+        # from ignoring the initial double quote to treating it as a phrase.
```

Comment:
To check: we've changed our parser to mirror this behaviour?

Reply:
Yes, so SQLite will match the behavior of PostgreSQL 14. PostgreSQL 11 - 13 will still use whatever their internal function call does. (And PostgreSQL 10 fallsback to `plainto_tsquery` anyway so let's just ignore talking about that.)